api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md

```diff
@@ -1,10 +1,182 @@
 ---
 id: dxGantt.Options.sorting
 type: dxGanttSorting
+inheritsType: dxGanttSorting
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures sorting settings.
+
+---
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/Sorting/"
+}
+
+Users can sort Gantt data by a single or multiple columns. Use the [mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) option to specify the sort mode. 
+
+- **Single Mode**. Click a column header to sort data by this column. Subsequent clicks on the same header reverse the sort order.
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-single.png)
+
+- **Multiple Mode**. Hold Shift and click a column header to sort data by multiple columns. Subsequent clicks on the same header reverse the sort order and clear sorting for the column. Set the [showSortIndexes](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes) option to **true** to show a column sort index within the header.
+
+    ![DevExtreme Gantt - Sorting](/images/Gantt/sorting-multiple.png)
+
+[note] To disable sorting for a particular column, set the column’s **allowSorting** option to **false**.
```

Comment:
I don't know how notes are rendered on the resulting page, but maybe you can write this sentences as plain text without note formatting.